new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDomainOnPV.java

```diff
@@ -0,0 +1,710 @@
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.operatorIsReady;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.podReady;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.serviceExists;
+import static oracle.weblogic.kubernetes.extensions.LoggedTest.logger;
+import static org.awaitility.Awaitility.with;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Tests to create domain on persistent volume using WLST and WDT.
+ */
+@DisplayName("Verify the domain can be created on persistent volume")
+@IntegrationTest
+public class ItDomainOnPV implements LoggedTest {
+
+  private static String opNamespace = null;
+  private static String domainNamespace = null;
+
+  private final String domainUid = "domain-onpv";
+  private final String clusterName = "pv-domain-cluster";
+  private final String adminServerName = "admin-server";
+  private final String adminServerPodName = domainUid + "-" + adminServerName;
+  private final String managedServerNameBase = "managed-server";
+  String managedServerPodNamePrefix = domainUid + "-" + managedServerNameBase;
+
+  private final int replicaCount = 2;
+
+  private final String pvName = domainUid + "-pv"; // name of the persistent volume
+  private final String pvcName = domainUid + "-pvc"; // name of the persistent volume claim
+
+  private String wlSecretName;
+  private final String adminUser = "system";
+  private final String adminPassword = "gumby1234";
+
+  // create standard, reusable retry/backoff policy
+  private static final ConditionFactory withStandardRetryPolicy
+      = with().pollDelay(2, SECONDS)
+          .and().with().pollInterval(10, SECONDS)
+          .atMost(5, MINUTES).await();
+
+  /**
+   * Install operator.
+   *
+   * @param namespaces injected by JUnit
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+
+    // get a unique operator namespace
+    logger.info("Get a unique namespace for operator");
+    assertNotNull(namespaces.get(0), "Namespace list is null");
+    opNamespace = namespaces.get(0);
+
+    // get a unique domain namespace
+    logger.info("Get a unique namespace for WebLogic domain");
+    assertNotNull(namespaces.get(1), "Namespace list is null");
+    domainNamespace = namespaces.get(1);
+
+    // install and verify operator
+    installAndVerifyOperator();
+  }
+
+  @Test
+  @DisplayName("Create domain in PV using WLST script")
+  public void testDomainOnPvUsingWlst() throws IOException {
+
+    // create pull secrets for WebLogic image
+    createOCRRepoSecret();
+
+    // create WebLogic credentials secret
+    createWebLogicCredentialsSecret();
+
+    // create persistent volume and persistent volume claims
+    createPVandPVC();
```

Comment:
we need to consider moving these methods into helper classes, it applies for MII methods also before the code is duplicated in multiple It classes as more tests will be added now.

Reply:
Will do in separate PR for all classes.